core/commands/bitswap.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var BitswapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "A set of commands to manipulate the bitswap agent.",
+		Tagline:          "Interact with the bitswap agent.",
```

Comment:
Not sure this is a useful change; bitswap isn't a command, it is a set of subcommands.